docs/metrics/bleu.md

```diff
@@ -0,0 +1,132 @@
+\end{align*}$$
+
+where the i-Gram precision is calculated as:
+
+$$
+p_i = \frac{\text{Clipped} \text{ count of matching i-grams in candidate text}^1}{\text{Total number of i-grams in candidate text}}
+$$
+
+Below, we explain the significance of the two components in the BLEU formula.
+??? info "n-Gram Overlap"
+    The **n-Gram Overlap** counts the number of 1-grams, 2-grams, 3-grams, and 4-grams of the output text that match the 1-, ..., 4-grams in the reference text — which is analogous to a precision score for the text. The 1-gram precision ensures that the correct vocabulary is used, whereas the 4-gram precision ensures that the candidate text is coherent.
+??? info "Brevity Penalty"
+    The **Brevity Penalty** is also applied to penalize the score for generating sentences that are less in length than the reference text. This is due to the fact that the n-Gram Overlap precision tends to give disproportionately high values to candidate texts that are very short in length, but contain most of the n-grams in the reference text.
+
+<div class="footnote-content">
+    <p style="font-size: smaller;">
+        <sup>1</sup> The <i>clipped count of matching i-grams in candidate text</i> is the minimum between the count of i-grams in the candidate text and the maximum count of i-grams in any of the reference texts for a given i-gram.
+    </p>
+</div>
+
+### Interpretation
+A known fact about BLEU scores is that they are not to be compared between different workflows and tasks. An excellent BLEU score for one task may be subpar for another. The score only serves as a general guideline to quantify the performance of your model — not to replace human judgement. That being said, what do these scores really mean, and how can we decipher them?
+
+1. **Higher is better**: Though BLEU scores may vary among tasks, one thing is for sure - higher scores are better. Generally speaking, a commonly accepted guideline is as follows: <br>
+
+    | Score | Qualitative Interpretation |
+    | ---   | ---                        |
+    | < 0.1 | Useless output             |
+    | 0.1 - 0.4 | Varies in quality; May not be acceptable |
+    | 0.4 - 0.6 | High quality generated text |
+    | > 0.6 | Better than human quality  |
+
+2. **Track Trends Over Time**: Rising scores signal improvements in models, while drops could hint at issues or changes in your dataset.
+
+3. **Combine It With Other Metrics**: BLEU primarily measures n-gram overlap, overlooking some nuancies like context and understanding. While a high BLEU score is promising, it doesn't guarantee flawless text. A complementary metric like [BertScore](bertscore.md) may help in quantifying your model's performance from other perspectives.
+
+## Example
+
+**Generated Sentence**: Kolena is an ML testing and debugging platform to find hidden model behaviors and demystify model development <br>
```

Comment:
How about using a table for a better display of these two sentences? 

<img width="796" alt="image" src="https://github.com/kolenaIO/kolena/assets/17771952/b02b821c-4d6d-450c-bbb6-b5395f14be05">

Reply:
```
| Generated | Reference |
| --- | --- |
| `Kolena is an ML testing and debugging platform to find hidden model behaviors and demystify model development` | `Kolena is a comprehensive machine learning testing and debugging platform to surface hidden model behaviors and take the mystery out of model development` |
```